react/features/video-status-label/components/VideoStatusLabel.js

```diff
@@ -136,54 +144,41 @@ export class VideoStatusLabel extends Component {
             = `${baseClasses} ${filmstrip} ${remoteVideosVisible} ${opening}`;
 
         return (
-            <div
```

Comment:
I want to comment on line 131 (a few lines above this one) that has no change in this pr:
displayedLabel = _largeVideoHD
                 ? t('videoStatus.hd') : t('videoStatus.sd');
As I said in another comment we should now take into account low quality here as well, which brings another question. It looks like _largeVideoHD as a boolean doesn't bring any value anymore, so we can check if we can replace it completely by the new receiveVideoQuality property from conference..WDYT?